src/router/index.js

```diff
@@ -1,6 +1,8 @@
 import Vue from 'vue';
 import VueRouter from 'vue-router';
 import AdminView from '../views/AdminView.vue';
+import AdminActive from '../components/Admin/AdminActive.vue'; // imported from components-Admin Active Reservations
```

Comment:
Comment not necessary
```suggestion
import AdminActive from '../components/Admin/AdminActive.vue'; 
```